### ISSUE TITLE
Show hex code in helm-ucs.

### ISSUE THREAD

**Issue by tmalsburg** [emacs-helm/helm#889]
helm-ucs shows the name of a character and the character itself.  In many situations it is also useful to know the hex code of the character.  Please consider adding that.

**Comment by thierryvolpiatto** [comment]
I have no idea how to do this, maybe you can enlight me, or even better provide a PR.

**Comment by tmalsburg** [comment]
Hi Thierry, please have a look at #892.

**Comment by michael-heerdegen** [comment]
Are we speaking here about the hex code of a character (in the sense that every character in Gnu Emacs is a number), or about the Unicode hex code?

Seems these are not always identical, since there are different read syntaxes for these two things.

**Comment by tmalsburg** [comment]
In this context, we'd like to see the unicode codepoint.  Are you suggesting that the results may depend on the locale under which Emacs is running? Looking at the source of `ucs-names`, which supplies the the data for `helm-ucs`, it seems to me that the results should be invariant to the locale.  I may be wrong, though.

**Comment by michael-heerdegen** [comment]
Titus von der Malsburg <notifications@github.com> writes:

> In this context, we'd like to see the unicode codepoint. Are you
> suggesting that the results may depend on the locale under which Emacs
> is running? Looking at the source of ucs-names, which supplies the the
> data for helm-ucs, it seems to me that the results should be invariant
> to the locale. I may be wrong, though.

I vaguely remember to have heard they can differ a long time ago.
Sadly, I do not know anything about this stuff.  We should be sure to
DTRT.

I don't think it is locate dependent however.  This is something
"outside of Emacs".

**Comment by tmalsburg** [comment]
I think it should be ok. `ucs-names` gives us the code points and all my code is doing is to convert them to hex.

**Comment by michael-heerdegen** [comment]
I see, mule does the critical work.

I could not find a case here were unicode hexcode and character number in hex would differ btw:

```
(loop for i from 1 to 1000000
      when (not (= (eval (read (format "?\\U00%06x" i)))
                   (eval (read (format "?\\x%06x" i)))))
      collect i)
==> nil
```

**Comment by tmalsburg** [comment]
Thanks for checking that, Michael!

**Comment by tmalsburg** [comment]
This issue was addressed in db55080767030cfdf78f1ae06171274b7f67c3f7 and 92b4506b16695d1e33dcb67c7070117692875afb.
